client/lib/media/utils/validate-media-item.js

```diff
@@ -8,7 +8,7 @@ import { isSupportedFileTypeInPremium } from 'calypso/lib/media/utils/is-support
  *
  * @param  {object}      site Site object
  * @param  {object}      item Media item
- * @returns {Array|null}      Validation errors, or null if no site.
+ * @returns {Array|undefined} Validation errors, or null if no site.
```

Comment:
We should alter the description here, as well. 
```suggestion
 * @returns {Array|undefined} Validation errors, or `undefined` if no site.
```